tests/testthat/test-compiler-ghpages.R

```diff
@@ -10,23 +22,38 @@ test_that("error for viz with no title", {
   viz.no.title <- viz
   viz.no.title$title <- NULL
   expect_error({
-    animint2pages(viz.no.title, "no-title")
+    animint2pages(viz.no.title, "no-title",server=s)
   }, "plot.list does not contain option named title, which is required by animint2pages")
 })
 
 test_that("error for viz with no source", {
   viz.no.source <- viz
   viz.no.source$source <- NULL
   expect_error({
-    animint2pages(viz.no.source, "no-source")
+    animint2pages(viz.no.source, "no-source",server=s)
   }, "plot.list does not contain option named source, which is required by animint2pages")
 })
 
 test_that("animint2pages() returns owner/repo string", {
-  viz_owner_repo <- animint2pages(viz, github_repo = "animint2pages_test_repo")
+  viz_owner_repo <- animint2pages(viz, github_repo = "animint2pages_test_repo",server=s)
```

Comment:
does the user have to provide server=s argument to get the screenshot to work? that would be inconvenient.
probably ok if only required to use server arg in tests.

Reply:
I will answer my own question: the user does not have to provide server=s argument to get the screenshot to work.
But then why is the server argument needed at all? please explain by writing documentation.

Reply:
> I will answer my own question: the user does not have to provide server=s argument to get the screenshot to work. But then why is the server argument needed at all? please explain by writing documentation.

server=s argument is used to pass persistent server object, which is ghpages test file in this case as that is the only file using animint2pages function. if the server object is not passed then the code creates a new server with a random port during every call to animint2pages function. I will  add this in wiki documentation to explain this.